packages/buildessential.rb

```diff
@@ -3,12 +3,17 @@
 class Buildessential < Package
   description 'A collection of tools essential to compile and build software.'
   homepage ''
-  version '1.0'
+  version '1.0-1'
 
   is_fake
 
   depends_on 'gcc'
   depends_on 'linuxheaders'
   depends_on 'make'
   depends_on 'pkgconfig'
+  
+  depends_on 'automake' #
+  depends_on 'libtool'  #
+  depends_on 'intltool' # These three are mostly required for "./autogen.sh"
```

Comment:
We might need `patch` and some other packages as well.  We should try recompiling a few more packages that require autogen.sh on a fresh Chromebrew install to confirm.  But yeah, this is essentially what I proposed in #1742.

Reply:
How about meson build?